Makefile

```diff
@@ -196,13 +196,10 @@ einterfaces-mocks: ## Creates mock files for einterfaces.
 pluginapi: ## Generates api and hooks glue code for plugins
 	$(GO) generate $(GOFLAGS) ./plugin
 
-check-licenses: ## Checks license status.
```

Comment:
Would you please add this to `build/legacy.mk` instead? This way we make sure, that the migration is smooth for devs.